lib/clean.js

```diff
@@ -38,8 +38,15 @@ function cleanup() {
 function cleanupCache() {
   const cacheLocation = getUserCachePath(this.options);
   if (fse.existsSync(cacheLocation)) {
+    let cleanupProgress;
     if (this.serverless) {
-      this.serverless.cli.log(`Removing static caches at: ${cacheLocation}`);
+      if (this.progress && this.log) {
```

Comment:
It's fine to check just for `this.log` (also later, when we may want to unconditionally switch to modern logs, it'll make easier to pick all those conditionals by searching `if (this.log`)